.travis.yml

```diff
@@ -1,6 +1,3 @@
 language: node_js
 node_js:
   - "7"
-  - "6"
```

Comment:
Yes, we are transpiling and bundling the source you can see [here](https://github.com/lycan-city/werewolf-brain/blob/master/package.json#L5) that we're not exposing the es6 version